File: java/easy/0001_two_sum.java
// https://leetcode.com/problems/two-sum/
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] locations = new int[2];
            for(int i = 0; i<nums.length-1; i++){
                for(int x = 1; x<nums.length; x++){
                    if(nums[i] + nums[x] == target && i != x){
                        locations[0] = i;
                        locations[1] = x;
                    }                    
                }

            }
        return locations;
    }
}

File: python/easy/0001_two_sum.py
# https://leetcode.com/problems/two-sum/description/
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res = []
        indexMap = {}
        for i in range(len(nums)):
            if nums[i] not in indexMap:
                indexMap[nums[i]] = i
            if target - nums[i] in indexMap and i != indexMap[target - nums[i]]:
                    res = [indexMap[target - nums[i]], i]
                    break
        return res
